src/webgpu/api/validation/encoding/cmds/compute_pass.spec.ts

```diff
@@ -240,7 +240,7 @@ g.test('indirect_dispatch_buffer,usage')
     });
     t.trackForCleanup(buffer);
 
-    const isValid = GPUBufferUsage.INDIRECT | bufferUsage;
+    const isValid = GPUBufferUsage.INDIRECT & bufferUsage;
```

Comment:
It looks like the original PR used the wrong bit operator. OR operator always sets true to |isValid|. But, dispatchWorkgroupsIndirect doesn't generate a validation error even when the buffer usage doesn't contain INDIRECT. So, it's likely that why the original PR did a mistake. IMO, we need to take a look at dawn or Chromium to find why dispatchWorkgroupsIndirect doesn't generate a validation error when using other usages.

Reply:
Nice catch! Sorry, it looks like I didn't review this test closely enough.

Calling dispatchWorkgroupsIndirect never generates a validation error. Command encoders are special - when there is an error in a command, it doesn't generate an error; instead it marks the encoder as failed, and the validation error doesn't occur until `.finish()` is called.

Since you are using `createEncoder()` here, you can simply use `validateFinish`. Example here:
https://github.com/gpuweb/cts/blob/a912567d309f6bde49e7e0c3b0671a2e138ab96c/src/webgpu/api/validation/encoding/cmds/render/setVertexBuffer.spec.ts#L35-L37

Reply:
No, it's my fault. I should have checked it. And, thank you for pointing the validateFinish out. I update the test to use validateFinish. Thanks!